### ISSUE TITLE
Evaluate model 

### ISSUE THREAD

**Issue by game-sys** [fizyr/keras-retinanet#1235]
how to evaluate retina-net model i just run the train.py and it saved model in snapshots i want to test those model on validation set. i am not getting how to give path to the folder of dataset i want to test

**Comment by game-sys** [comment]
and how it calculate this mAP on training time is it mAP against traival.txt data or testing.txt
318 instances of class bus with average precision: 0.8337
1119 instances of class car with average precision: 0.8606
376 instances of class motorbike with average precision: 0.8620
mAP: 0.8521

**Comment by tonmoyborah** [comment]
You can run `keras_retinanet/bin/evaluate.py` to evaluate on any valid dataset. You can pass many arguments to `evaluate.py` like
 --backbone csv /path/to/csv_annotation_file  /path/to/class_names_file   /path/to/model_file

**Comment by hgaiser** [comment]
Seems this has been answered.
